packages/web/src/components/nav/desktop/WalletsNestedContent.tsx

```diff
@@ -37,6 +38,8 @@ export const WalletsNestedContent = () => {
   const audioBalance = useTotalBalanceWithFallback()
   const userId = useSelector(getUserId)
   const { tier } = useSelectTierInfo(userId ?? 0)
+  const usdcCentBalance =
+    formatUSDCWeiToFloorCentsNumber((usdcBalance ?? new BN(0)) as BNUSDC) / 100
```

Comment:
a bit code smell.. feels like the `/ 100` should be in the format fn, and `useUSDCBalance` should return `BN(0)` if it fails to fetch. but can address later.